libpysal/graph/tests/test_builders.py

```diff
@@ -1,8 +1,12 @@
-import pytest
-
+import geodatasets
 import geopandas as gpd
+import numpy as np
 import pandas as pd
-import geodatasets
+import pytest
+from numpy.testing import assert_array_almost_equal
+from scipy.sparse.csr import csr_matrix
```

Comment:
```suggestion
from scipy.sparse import csr_matrix
```
Otherwise it raises ```DeprecationWarning: Please import `csr_matrix` from the `scipy.sparse` namespace; the `scipy.sparse.csr` namespace is deprecated and will be removed in SciPy 2.0.0.``` in CI.